src/Backend/Modules/CommerceMollie/Domain/PaymentMethod/MolliePaymentMethodType.php

```diff
@@ -0,0 +1,181 @@
+                    'class' => OrderStatus::class,
+                    'query_builder' => function (EntityRepository $er) {
+                        return $er->createQueryBuilder('i')
+                            ->orderBy('i.title', 'ASC');
+                    },
+                    'choice_label' => 'title',
+                ]
+            )
+            ->add(
+                'orderCancelledId',
+                EntityType::class,
+                [
+                    'required' => true,
+                    'label' => 'lbl.OrderCancelled',
+                    'class' => OrderStatus::class,
+                    'query_builder' => function (EntityRepository $er) {
+                        return $er->createQueryBuilder('i')
+                            ->orderBy('i.title', 'ASC');
+                    },
+                    'choice_label' => 'title',
+                ]
+            )
+            ->add(
+                'orderExpiredId',
+                EntityType::class,
+                [
+                    'required' => true,
+                    'label' => 'lbl.OrderExpired',
+                    'class' => OrderStatus::class,
+                    'query_builder' => function (EntityRepository $er) {
+                        return $er->createQueryBuilder('i')
+                            ->orderBy('i.title', 'ASC');
+                    },
+                    'choice_label' => 'title',
+                ]
+            )
+            ->add(
+                'orderRefundedId',
+                EntityType::class,
+                [
+                    'required' => true,
+                    'label' => 'lbl.OrderRefunded',
+                    'class' => OrderStatus::class,
+                    'query_builder' => function (EntityRepository $er) {
+                        return $er->createQueryBuilder('i')
+                            ->orderBy('i.title', 'ASC');
+                    },
+                    'choice_label' => 'title',
+                ]
+            );
+
+        $builder
+            ->get('orderInitId')
+            ->addModelTransformer(new OrderStatusTransformer($options['entityManager']));
+        $builder
+            ->get('orderCompletedId')
+            ->addModelTransformer(new OrderStatusTransformer($options['entityManager']));
+        $builder
+            ->get('orderCancelledId')
+            ->addModelTransformer(new OrderStatusTransformer($options['entityManager']));
+        $builder
+            ->get('orderExpiredId')
+            ->addModelTransformer(new OrderStatusTransformer($options['entityManager']));
+        $builder
+            ->get('orderRefundedId')
+            ->addModelTransformer(new OrderStatusTransformer($options['entityManager']));
+
+        foreach ($options['enabledMethods'] as $method) {
+            /**
+             * @var FormFactory $formFactory
+             */
+//            $formFactory = Model::get('form.factory');
```

Comment:
Should this be still commented out? Or can it be removed?

Reply:
I re-used this file from the catalog module, this was commented there but I guess it can be removed.